topcoffea/modules/corrections.py

```diff
@@ -71,9 +73,12 @@ def btag_sf_eval(jet_collection,wp,year,method,syst):
     pt_flat = ak.where(pt_flat>1000.0,1000.0,pt_flat)
 
     # Evaluate the SF
-    ceval = correctionlib.CorrectionSet.from_file(fname)
-    sf_flat = ceval[method].evaluate(syst,wp,flav_flat,abseta_flat,pt_flat)
-    sf = ak.unflatten(sf_flat,ak.num(jet_collection.pt))
+    if is_run3:
+        sf = ak.ones_like(jet_collection.pt)
+    elif is_run2:
+        ceval = correctionlib.CorrectionSet.from_file(fname)
+        sf_flat = ceval[method].evaluate(syst,wp,flav_flat,abseta_flat,pt_flat)
+        sf = ak.unflatten(sf_flat,ak.num(jet_collection.pt))
```

Comment:
Looks like you have some old commits. Can you merge in the latest master branch changes?

Reply:
Thanks, looks good!